repose-aggregator/commons/commons-utilities/src/main/java/org/openrepose/commons/utils/http/media/MimeType.java

```diff
@@ -24,6 +24,8 @@
 @SuppressWarnings("PMD.AvoidDuplicateLiterals")
 public enum MimeType {
 
+    // TODO: This list isn't even close to being exhaustive.
+    //       An enum is probably not the right structure for this data since it cannot represent unlisted MIME types.
```

Comment:
So I stopped using this class in the Rate Limiting filter, at least. I switch to Spring Web's MediaType instead.